Packs/VirusTotal/Integrations/FeedLivehunt/FeedLivehunt.py

```diff
@@ -0,0 +1,315 @@
+import urllib3
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+from datetime import datetime
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+class DetectionRatio:
+    malicious = 0
+    total = 0
+
+    def __init__(self, last_analysis_stats: dict):
+        self.malicious = last_analysis_stats['malicious']
+        self.total = last_analysis_stats['harmless'] + \
+            last_analysis_stats['suspicious'] + \
+            last_analysis_stats['undetected'] + \
+            last_analysis_stats['malicious']
+
+    def __repr__(self):
+        return f'{self.malicious}/{self.total}'
+
+
+class Client(BaseClient):
+    def get_api_indicators(self,
+                           query_filter: Optional[str] = None,
+                           limit: Optional[int] = 10):
+        return self._http_request(
+            'GET',
+            'intelligence/hunting_notification_files',
+            params=assign_params(
+                filter=query_filter,
+                limit=min(limit, 40),
+            )
+        )
+
+    def fetch_indicators(self,
+                         limit: Optional[int] = 10,
+                         filter_tag: Optional[str] = None,
+                         fetch_command: bool = False) -> List:
+        """Retrieves all entries from the feed.
+        Returns:
+            A list of objects, containing the indicators.
+        """
+        result = []
+        query_filter = ''
+
+        if isinstance(filter_tag, str):
+            query_filter = f'tag:"{filter_tag}"'
+
+        if fetch_command:
+            if last_run := self.get_last_run():
+                query_filter = f'{query_filter} {last_run}'
+
+        response = self.get_api_indicators(query_filter, limit)
+
+        try:
+            for indicator in response.get('data', []):
+                result.append({
+                    'data': indicator,
+                    'type': 'file',
+                    'FeedURL': self._base_url
+                })
+        except ValueError as err:
+            demisto.debug(str(err))
+            raise ValueError(f'Could not parse returned data as indicator. \n\nError message: {err}')
+
+        self.set_last_run()
```

Comment:
If the fetch_command arg is 'False' (test_module) for example, do we still want to set the last run?

Reply:
You are right, set last run isn't necessary when `fetch_command` is false. Good catch! Fixed.